interface/modules/custom_modules/oe-module-faxsms/src/Controller/AppDispatch.php

```diff
@@ -619,4 +625,12 @@ public function verifyAcl($sect = 'patients', $v = 'docs', $u = ''): bool
     {
         return AclMain::aclCheckCore($sect, $v, $u);
     }
+
+    /**
+     * @return null
+     */
+    protected function index()
```

Comment:
This was invoked as part of the `dispatchActions()`.  I didn't take the time to figure it out, but implementations in other sub classes didn't seem to do anything, so I thought it would simplify new providers if there was a worked default to use.

Reply:
I believe index was meant for the API use but unsure.